src/app/components/card/card.component.ts

```diff
@@ -36,48 +35,61 @@ export interface CardAction {
     action?: Action;
 }
 
+export interface CardRights {
+    name: string;
+    in: boolean;
```

Comment:
What does the in means ?

Reply:
'in' its an from the old code. Its to know when the user is in a group or not.